TipCatDotNet.Api/Controllers/MemberController.cs

```diff
@@ -44,6 +44,27 @@ public async Task<IActionResult> Add([FromRoute] int accountId, [FromBody] Membe
         }
 
 
+        /// <summary>
+        /// Adds a member to a facility into an account.
+        /// </summary>
+        /// <param name="accountId">Target account ID</param>
+        /// <param name="facilityId">Target facility ID</param>
+        /// <param name="memberRequest">Change request</param>
+        /// <returns></returns>
+        [HttpPost("accounts/{accountId}/facilities/{facilityId}/members")]
```

Comment:
It's more convenient when you build routes another way: first a subject, and a target action after that.
`accounts/{accountId}//members/{memberId}/facilities/{facilityId}/transfer` or `accounts/{accountId}//members/{memberId}/transfer/facilities/{facilityId}`.
Also, no need to pass whole member model here.